release-notes/EPM/I-DOCSIS/EPM_7.0.1_I-DOCSIS.md

```diff
@@ -0,0 +1,72 @@
+---
+uid: EPM_7.0.1_I-DOCSIS
+---
+
+# EPM 7.0.1 I-DOCSIS
+
+> [!IMPORTANT]
+> We are still working on this release. Some release notes may still be modified or moved to a later release. Check back soon for updates!
+
+## New features
+
+#### Channel Utilization added for US/DS Service Group [ID_38849]
+
+When you open the Upstream or Downstream Service Group levels under the DOCSIS Service Group chain in the Topology app, the Channel Utilization average of the associated channels is now displayed in the data section.
+
+#### Skyline EPM Platform: New default values for US/DS SNR threshold [ID_39054]
+
+New default values have been added for the upstream and downstream SNR threshold:
+
+- For the upstream SNR threshold:
+
+  - "Other" Modulation: 20.0 dB
+  - QPSK: 15.0 dB
+  - QAM8: 15.0 dB
+  - QAM16: 15.0 dB
+  - QAM32: 19.0 dB
+  - QAM64: 21.0 dB
+  - QAM128: 24.0 dB
+
+- For the downstream SNR threshold:
+
+  - "Other" Modulation: 20.0 dB
+  - QAM64: 15.0 dB
+  - QAM256: 27.0 dB
+
+When the Skyline EPM Platform element is first initialized, it will now set default values for the Upstream QAM Channel Threshold and Downstream QAM Channel Threshold. In case the element already has values in the tables, you can add the default values with a button. This button is located on the Thresholds Settings page next to the *Execute* button.
+
+#### New time picker for QAM DS and US Channels dashboards [ID_39074]
+
+In the *Node Segment* and *Service Groups* folders in the Dashboards app, a new feature is available on the *QAM DS Channels* and *QAM US Channels* pages, which allows you to select specific start and end times for the information shown.
+
+To use this feature, click the timestamp at the top of the dashboard. This will open a window where you can select the desired start and end times. When you do so, the trend graph will automatically update to display the information for the selected time period. To close the pop-up window, click outside of the window.
+
```

Comment:
Hi @SkylineGBO, checking the vnedor's page, the name of the product is CableOS. Please could we update the name?

Reply:
I have updated the RN